Error running v0.13.1

Installing `knip@0.13.1` with `yarn` prints the following:

```
YN0002: │ knip@npm:0.13.1 doesn't provide esbuild (peefd7), requested by esbuild-register
```

And as expected, it fails to run:

```
node:internal/modules/cjs/loader:995
  const err = new Error(message);
              ^

Error: Cannot find module 'esbuild'
Require stack:
- /home/dev/knip-test/node_modules/esbuild-register/dist/node.js
    at Module._resolveFilename (node:internal/modules/cjs/loader:995:15)
    at Module._load (node:internal/modules/cjs/loader:841:27)
    at Module.require (node:internal/modules/cjs/loader:1061:19)
    at require (node:internal/modules/cjs/helpers:103:18)
    at Object.<anonymous> (/home/dev/knip-test/node_modules/esbuild-register/dist/node.js:4622:16)
    at Module._compile (node:internal/modules/cjs/loader:1159:14)
    at Module._compile (/home/dev/knip-test/node_modules/pirates/lib/index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1213:10)
    at Object.newLoader [as .js] (/home/dev/knip-test/node_modules/pirates/lib/index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:1037:32) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/dev/knip-test/node_modules/esbuild-register/dist/node.js'
  ]
}
```

--- a/package.json
+++ b/package.json
@@ -44,6 +44,7 @@
   "dependencies": {
     "@snyk/github-codeowners": "1.0.0",
     "easy-table": "1.2.0",
+    "esbuild": "0.15.13",
     "esbuild-register": "3.3.3",
     "globby": "13.1.2",
     "micromatch": "4.0.5",

